client/app/reducers/dispute.reducer.js

```diff
@@ -0,0 +1,51 @@
+export function disputeReducer(state = {}, action) {
+  const dispatch = action.type;
+  let newState = {};
+  if (dispatch === 'newDispute') {
+    let dispute;
+    fetch('/disputes', {
+      method: 'GET',
+      credentials: 'include',
+      headers: {
+        Accept: 'application/json'
+      }
+    })
+    .then(dataStream => dataStream.json())
+    .then(resp => {
+      newState = resp[Math.floor(Math.random()*resp.length)];
```

Comment:
Can we do the random selection on the back end and only give 1 dispute to the client?